calico_containers/pycalico/util.py

```diff
@@ -71,3 +75,8 @@ def get_host_ips(version=4, exclude=None):
 
     return ip_addrs
 
+def get_hostname():
+    try:
```

Comment:
Can you comment this. Both a docstring and the reason why it falls back to the socket.gethostname()